pkg/bench/rttanalysis/testdata/benchmark_expectations

```diff
@@ -31,19 +31,19 @@ exp,benchmark
 23,CreateRole/create_role_with_2_options
 24,CreateRole/create_role_with_3_options
 22,CreateRole/create_role_with_no_options
-22,DropDatabase/drop_database_0_tables
-31,DropDatabase/drop_database_1_table
-40,DropDatabase/drop_database_2_tables
-49,DropDatabase/drop_database_3_tables
+21,DropDatabase/drop_database_0_tables
+28,DropDatabase/drop_database_1_table
+35,DropDatabase/drop_database_2_tables
+42,DropDatabase/drop_database_3_tables
```

Comment:
It appears that allowing most system tables to be leased can lead to some significant roundtrip savings. I'm guessing this is the cause, at least (not sure what else it could be) and for drops I'm guessing it specifically involves leasing the jobs table.